<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Acivity</class>
 <widget class="QWidget" name="Widget_Acivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="edit_Name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Acivity">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>70</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>添加活动</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_building">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>120</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>校区</string>
   </property>
   <widget class="QRadioButton" name="radio_IsXiTuCheng">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>西土城校区</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_IsShaHe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>沙河校区</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_acivity">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>活动类型</string>
   </property>
   <widget class="QRadioButton" name="radio_ForSingal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>个人活动</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_ForPeople">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>集体活动</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_startHour">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>120</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_endHour">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>120</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>991</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_single">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询个人活动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_allPeople">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询集体活动</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_timePaixu">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>时间排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_namePaixu">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>名称排序</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_labelSelect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>活动名称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_event">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_eventDelete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>740</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>活动删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
